.github/ISSUE_TEMPLATE/als-----wil-ik----.md

```diff
@@ -26,6 +26,7 @@ about: Beschrijving van een feature/user story
 - [ ] de DSO URI- en API-strategie worden gevolgd of afwijkingen zijn vastgelegd als ontwerp keuze
 - [ ] eventueel gemaakte ontwerp keuzes zijn gedocumenteerd
 - [ ] er zijn geen (nieuwe) conflicten met ontwerp keuzes in [BIP](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/blob/master/docs/design_decisions.md)
+- [ ] de gewenste (aanvulling op de) functionaliteit (van de API's) is geanalyseerd en beschreven.
```

Comment:
Dit hoort bij de Definition of Ready. Is de uitwerking van de userstory naar wat er gebouwd zou moeten worden. In de realisatie-sprint wordt bepaald hoe dat gaat plaatsvinden en wordt het daadwerkelijk gebouwd.